### PR TITLE
Batch release prepare

### PR DESCRIPTION
**Pull request by st0012**
First part of #1449

### CODE REVIEW

File: .craft.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+github:
+    owner: getsentry
+    repo: sentry-ruby
+changelogPolicy: simple
+preReleaseCommand: ruby .scripts/batch_release.rb
+releaseBranchPrefix: batch-release-sentry-ruby
+statusProvider:
+  name: github
+artifactProvider:
+  name: github
```
Comment (BYK) [inline .craft.yml:11]: These are now the defaults when we set `0.22.2` as the min version. (well, anything >=0.21.0 actually https://github.com/getsentry/craft/releases/tag/0.21.0)

File: .craft.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+targets:
```
Comment (BYK) [inline .craft.yml:12]: No `github` target?

File: .scripts/batch_release.rb [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+def update_gemspec_dependency(gem_name, version)
+  file_name = "#{gem_name}/#{gem_name}.gemspec"
+  text = File.read(file_name)
+  new_contents = text.gsub(/spec.add_dependency "sentry-ruby-core", ".+"/, "spec.add_dependency \"sentry-ruby-core\", \"~> #{version}\"")
+  File.open(file_name, "w") {|file| file.puts new_contents }
+end
+
+# when craft runs scripts it inserts version to the 2nd argument
+version = ARGV[1]
```
Comment (BYK) [inline .scripts/batch_release.rb:30]: It also sets environment variables if that's easier:

https://github.com/getsentry/craft/blob/5030c50f8c37e81f39ce51c4a419ec787cc8f774/src/commands/prepare.ts#L249-L250

We may wanna document this.

**Comment by BYK** [comment]
Did not approve due to lack of `github` target. Once we clarify that, I'll approve as the rest looks good.

File: .scripts/batch_release.rb (hunk 3 above)
Comment (st0012) [reply .scripts/batch_release.rb:30]: @BYK thanks for the tip. I'll change this later with the other version bumping script 👍

File: .craft.yml [hunk 4]
```diff
@@ -0,0 +1,29 @@
+minVersion: '0.22.2'
+github:
+    owner: getsentry
+    repo: sentry-ruby
```
Comment (BYK) [inline .craft.yml:4]: You can remove these too, they are now inferred automatically.
```suggestion
```

**Review by BYK — APPROVED** [review]

**Pull request merged**